### PR TITLE
Ent 789 teams it license

### PR DESCRIPTION
**Pull request by aaronbini**
## What are you changing in this pull request and why?

Some minor docs updates to reflect changes in the new IT License.

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `website/static/_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/static/_redirects [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/docs/cloud/manage-access/licenses-and-groups /docs/cloud/manage-access/about-user-access 301
```
Comment (mirnawong1) [inline website/static/_redirects:1]: adding bc i think we can delete the `licenses and groups` file

File: website/docs/docs/cloud/manage-access/about-access.md [hunk 2]
```diff
@@ -31,14 +31,12 @@ user can only have one type of license at any given time.
 
 A user's license type controls the features in dbt Cloud that the user is able
 to access. dbt Cloud's three license types are:
- - **Read Only**
- - **Developer**
- - **IT**
+
+ - **Developer** &mdash; User may be granted _any_ permissions.
```
Comment (mirnawong1) [inline website/docs/docs/cloud/manage-access/about-access.md:35]: consolidating paragraph into bullets so its easier to scan and less reading for user

**Review by mirnawong1 — APPROVED** [review]
lgtm!

File: website/docs/docs/cloud/manage-access/cloud-seats-and-users.md [hunk 3]
```diff
@@ -6,21 +6,22 @@ sidebar: "Users and licenses"
...
+| Functionality | Developer User | Read Only Users | IT Users* |
 | ------------- | -------------- | --------------- | -------- |
 | Use the Developer IDE | ✅ | ❌ | ❌ |
 | Use Jobs | ✅ | ❌ | ❌ |
 | Manage Account | ✅ | ❌ | ✅ |
 | API Access | ✅ | ❌ | ❌ |
 | Use [Source Freshness](/docs/deploy/source-freshness) | ✅ | ✅ | ❌ |
 | Use [Docs](/docs/collaborate/build-and-view-your-docs) | ✅ | ✅ | ❌ |
-
+*Available on Enterprise and Team plans only.
```
Comment (asmclean) [inline website/docs/docs/cloud/manage-access/cloud-seats-and-users.md:24]: Is this intended to be rendered as a row in the table?  That's what I'm seeing when I view the rendered Markdown.

**Review by asmclean — APPROVED** [review]

File: website/docs/docs/cloud/manage-access/cloud-seats-and-users.md (hunk 3 above)
Comment (mirnawong1) [reply website/docs/docs/cloud/manage-access/cloud-seats-and-users.md:24]: it's looking like this to me, below the table:

Screenshot 2023-06-28 at 15 58 57

Comment (mirnawong1) [reply website/docs/docs/cloud/manage-access/cloud-seats-and-users.md:24]: is that what you mean @asmclean ?

Comment (asmclean) [reply website/docs/docs/cloud/manage-access/cloud-seats-and-users.md:24]: Ah, okay.  I was just looking at the way Github rendered it, which doesn't match.  (Which isn't a huge surprise.)  I didn't think what I was seeing was correct.  Thanks for confirming.

Screenshot 2023-06-28 at 11 05 54 AM

**Review by asmclean — APPROVED** [review]

File: website/docs/docs/cloud/manage-access/cloud-seats-and-users.md (hunk 3 above)
Comment (mirnawong1) [reply website/docs/docs/cloud/manage-access/cloud-seats-and-users.md:24]: sounds great! i'll go ahead and merge now, thank you again!

**Comment by mirnawong1** [comment]
hey @asmclean is it true that an IT license does not count toward seat usage for enterprise and team plans?

**Pull request merged**